tasks/migrations/0003_user_test_tasks.py

```diff
@@ -0,0 +1,127 @@
+        ('users', '0002_user_test_data'),
+    ]
+
+    def generate_data(apps, schema_editor):
+        from tasks.models import Task, Priority, Status, Comment
+        from users.models import User
+
+        with transaction.atomic():
+            employee_1 = User.objects.get(user__username="emp1")
+            employee_11 = User.objects.get(user__username="emp11")
+            employee_2 = User.objects.get(user__username="emp2")
+            manager_1 = User.objects.get(user__username="man1")
+            manager_2 = User.objects.get(user__username="man2")
+
+            task_11 = Task.objects.create(title="task 11",
+                                          assignee=employee_1,
+                                          created_by=manager_1,
+                                          priority=Priority.HIGH,
+                                          status=Status.IN_PROGRESS,
+                                          description="build me a house")
+
+            task_12 = Task.objects.create(title="task 12",
+                                          assignee=employee_11,
+                                          created_by=manager_1,
+                                          priority=Priority.HIGH,
+                                          status=Status.IN_PROGRESS,
+                                          description="sing me a song")
+
+            task_13 = Task.objects.create(title="task 13",
+                                          assignee=employee_1,
+                                          created_by=manager_1,
+                                          priority=Priority.HIGH,
+                                          status=Status.IN_PROGRESS,
+                                          description="task 13 description")
+
+            task_14 = Task.objects.create(title="task 14",
+                                          assignee=employee_11,
+                                          created_by=manager_1,
+                                          priority=Priority.HIGH,
+                                          status=Status.IN_PROGRESS,
+                                          description="task 14 description")
+
+            task_21 = Task.objects.create(title="task 21",
+                                          assignee=employee_2,
+                                          created_by=manager_2,
+                                          priority=Priority.HIGH,
+                                          status=Status.IN_PROGRESS,
+                                          description="task 21 description")
+
+            task_22 = Task.objects.create(title="task 22",
+                                          assignee=employee_2,
+                                          created_by=manager_2,
+                                          priority=Priority.HIGH,
+                                          status=Status.IN_PROGRESS,
+                                          description="task 22 description")
```

Comment:
maybe you can create the tasks with a loop instead of repeating this code
you can see kind of examples in:
https://github.com/redhat-beyond/ClassRater/blob/main/homepage/migrations/0003_course_test_data.py#L14
https://github.com/redhat-beyond/ClassRater/blob/main/homepage/migrations/0002_user_test_data.py#L14